### PR TITLE
Landing / sign_in merge

### PR DESCRIPTION
**Pull request by BWieber**
Hi Kevin, 

Woo - more Rails work! I added the initial stylings, but and I will adjust to make sure we don't have duplicate logins (there is a sign in option still in the top right). 

Here is what it looks like : 

landing_signin_merge

### CODE REVIEW

**Comment by kevinrobinson** [comment]
@BWieber thanks for talking this out and learning me about Devise!

**Comment by BWieber** [comment]
Hi Kevin! 

I finished up the sign in / landing page merge. Take a look : 

landing_merge_final

Devise was being a bit particular about how it wanted its routes set, so I relented. Everything is in looking order, and I gave the button some style points. Let me know if it needs anything else! 
 
_Brandon_

File: app/views/pages/about.html.erb [hunk 1]
```diff
@@ -1,6 +1,28 @@
 <div class="info-area">
   <div class="about-page detailed-content-area">
 
+    <div class="info-area">
```
Comment (kevinrobinson) [inline app/views/pages/about.html.erb]: We can remove this, right?

File: config/routes.rb [hunk 2]
```diff
@@ -6,7 +6,10 @@
     root to: 'educators#homepage', as: 'educator_homepage'
   end
 
-  root 'pages#about'
+  devise_scope :educator do
+    root to: "devise/sessions#new"
+  end
+
   get 'about' => 'pages#about'
```
Comment (kevinrobinson) [inline config/routes.rb]: Will we be able to remove this too?

File: db/schema.rb [hunk 3]
```diff
@@ -188,11 +188,10 @@
...
     t.integer  "student_id"
-    t.integer  "provided_by_educator_id"
-    t.integer  "recorded_by_educator_id"
+    t.integer  "educator_id"
     t.integer  "service_type_id"
+    t.text     "text"
     t.datetime "recorded_at"
-    t.datetime "date_started"
     t.datetime "created_at"
     t.datetime "updated_at"
```
Comment (kevinrobinson) [inline db/schema.rb]: good practice opportunity for fixing this up again :)

**Comment by kevinrobinson** [comment]
@BWieber This is great, thanks for your help!

One other piece of design feedback is that this changes the feel of the landing page a bit (for folks who are not signing in to use the product).  I think this is an improvement already, but I'd be interested in seeing if we can compress the whitespace at the top or make it more obvious at the top of the page about what this project is.

I'd suggest finishing this work first though, and we can tackle that after.

**Comment by alexsoble** [comment]
Really nice UI improvement @BWieber!

**Comment by BWieber** [comment]
Hey Kevin, 

I took a stab a different styling, and it definitely feels a bit better. It's hard to capture in a screen shot, so here are two to get a look: 

new_merge_style

Scrolling...

new_merge_2

Let me know if this looks good to you !

**Comment by BWieber** [comment]
Hey Kevin, 

Here is another design. I killed out the whitespace and horizontally aligned the log in elements. Go Flexbox! 

top_center_merge

This seems the most intuitive, and it doesn't hog the page in anyway. Let me know which direction you like best and we'll make it that way :)

**Comment by BWieber** [comment]
Oh - and I'll get the Capybara tests fixed too !

**Comment by kevinrobinson** [comment]
@BWieber Nice, yay flexbox!  I think this is a great improvement - it removes the hop for regular users between the about page and the sign in page, but it still keeps the copy explaining the project for people who come here from Google or the GitHub page.

This does look a little compressed, so one other thought might be to pull the sign in fields up above the gray line break into the header.  Alternately a border with some small margin around the sign in fields might help it feel a little cohesive, versus lost in the marketing copy.  No strong feelings either way, just some thoughts :+1:

**Comment by BWieber** [comment]
Hey Kevin, 

Added an inset border to it doesn't get lost :) 

Will work on Capybara tests to follow :

landing page - inset border

**Comment by BWieber** [comment]
Tests passing! Anything else you want before a :ship: ?

**Comment by alexsoble** [comment]
Hi Brendan,

:tada: for the passing tests! I remember when this PR used to show up :x: on Travis!

I reviewed the PR. I see three small issues that @kevinrobinson mentioned here and as [in-line notes](https://github.com/studentinsights/studentinsights/pull/198/files) that would be good to fix up before we merge:

1. This PR contains changes to `schema.rb`. The changes don't have to do with the login flow, let's remove before we merge.
2. As Kevin pointed out, we can remove `about.html.erb`. It's unused and all of the content has been moved to `devise/sessions/new.html.erb`.
3. I agree with Kevin that the login fields look compressed floating between the navbar and the main content area. I think they'd look slicker up top in the navbar. Plus there's plenty of whitespace up there now on the login page.

Thanks for all the work you've put in @BWieber! Tackle these three fixes and we will be ready to :ship: :shipit:!

**Comment by BWieber** [comment]
Hi Kevin, Alex, 

I have positioned the sign in bar in the upper division of the landing page. As you imagined so astutely, it looks great up there, and gives the page a clean feel: 

screen shot 2016-04-12 at 8 41 24 pm

I'll head back and remove unnecessary code files. Let me know if you'd like any other styling here! 

_Brandon_

File: spec/support/educator_sign_in_helpers.rb [hunk 4]
```diff
@@ -5,7 +5,6 @@ def educator_sign_in(educator)
 
 def sign_in_attempt(email, password)
   visit root_url
-  click_link 'Sign In'
```
Comment (kevinrobinson) [inline spec/support/educator_sign_in_helpers.rb]: @alexsoble are there Capybara specs here?  I never realized.

Comment (alexsoble) [inline spec/support/educator_sign_in_helpers.rb]: @kevinrobinson There are! Consider it realized!  :P

Comment (alexsoble) [inline spec/support/educator_sign_in_helpers.rb]: @kevinrobinson There are just two, they are in `spec/features`

Comment (alexsoble) [inline spec/support/educator_sign_in_helpers.rb]: @kevinrobinson And they don't test any JS features, just HTML and CSS

**Pull request merged**